samples/layouts/stepper/stepper-overview-example/index.html

```diff
@@ -0,0 +1,190 @@
+                  </igc-card-media>
+                  <igc-card-content>
+                    <span class="card-currency">400</span>
+                    <span class="card-title">STATEMENT CREDIT OFFER</span>
+                    <p>Business Customized Advanced</p>
+                    <p>Cash Mastercard</p>
+                  </igc-card-content>
+              </igc-card>
+            </div>
+          </div>
+          <div class="card-wrapper">
+            <igc-badge>Your current choice</igc-badge>
+            <igc-card class="card-first-step">
+              <div class="card-content-wrapper">
+                <igc-card-media>
+                  <img src="https://www.infragistics.com/angular-demos/assets/images/stepper/card-red.png" />
+                </igc-card-media>
+                <igc-card-content>
+                  <span class="card-currency">600</span>
+                  <span class="card-title">STATEMENT CREDIT OFFER</span>
+                  <p>Business Travel Advanced</p>
+                  <p>World Mastercard</p>
+                </igc-card-content>
+              </div>
+            </igc-card>
+          </div>
+          <div class="card-wrapper">
+            <igc-badge>Your current choice</igc-badge>
+            <igc-card class="card-first-step">
+              <div class="card-content-wrapper">
+                <igc-card-media>
+                  <img src="https://www.infragistics.com/angular-demos/assets/images/stepper/card-gold.png" />
+                </igc-card-media>
+                <igc-card-content>
+                  <span class="card-currency">500</span>
+                  <span class="card-title">STATEMENT CREDIT OFFER</span>
+                  <p>Business Golden</p>
+                  <p>World Mastercard</p>
+                </igc-card-content>
+              </div>
+            </igc-card>
+          </div>
+    </div>
+    </igc-step>
+    
+    <igc-step invalid>
+      <span slot="title">Business Information</span>
+      <div class="sample-row">
+        <igc-form class="flex-form">
+          <igc-input required label="Name" type="text" name="name"></igc-input>
+          <igc-input required label="Physical address" type="text" name="physicalAddress"></igc-input>
+          <div class="form-row">
+            <igc-input required label="City" type="text" name="city"></igc-input>
+            <igc-select label="State" name="state" required>
+              <igc-select-item value="nevada">Nevada</igc-select-item>
+              <igc-select-item value="newYork">New York</igc-select-item>
+              <igc-select-item value="newJersey">New Jersey</igc-select-item>
+              <igc-select-item value="ohio">Ohio</igc-select-item>
+              <igc-select-item value="texas">Texas</igc-select-item>
+            </igc-select>
+            <igc-mask-input required label="ZIP Code" type="text" name="zip">
```

Comment:
The length of the zip code must be 5 digits in order to be considered valid

Reply:
It will be considered valid only when its lenght is 5 digits. However, there is an [issue](https://github.com/IgniteUI/igniteui-webcomponents/issues/558) with the Mask Input component, that does not allow the highlight color of the input filed to be changed when the invalid state is changed.